openshift/Jenkinsfile.ocp4

```
@@ -47,12 +47,12 @@ pipeline {
                     sh "CPU_REQUEST=500m CPU_LIMIT=500m MEMORY_REQUEST=2Gi MEMORY_LIMIT=2Gi REPLICAS=3 PROJ_TARGET=${projProd} PROJ_TOOLS=e1e498-tools POSTGRES_USER=wps POSTGRES_DATABASE=wps IMAGE_REGISTRY=image-registry.openshift-image-registry.svc:5000 SECOND_LEVEL_DOMAIN=apps.silver.devops.gov.bc.ca POSTGRES_WRITE_HOST=patroni-wps-prod-leader POSTGRES_READ_HOST=patroni-wps-prod-replica ./openshift/scripts/oc_deploy.sh prod apply"
                     // TODO: Apply these additional steps:
                     // // Env Canada Subscriber
```

Pedantic, but can you drop the double //?
```suggestion
                    // Env Canada Subscriber
```